src/coreclr/tools/aot/ILCompiler.ReadyToRun/Compiler/DependencyAnalysis/ReadyToRun/InstanceEntryPointTableNode.cs

```diff
@@ -53,7 +53,7 @@ public override void AppendMangledName(NameMangler nameMangler, Utf8StringBuilde
 
         public static byte[] BuildSignatureForMethodDefinedInModule(MethodDesc method, NodeFactory factory)
```

Comment:
InstanceEntryPointTableNode seems to use this to build a hashtable of all methods stored in a R2R module. I would expect that we want to capture whether the method is the async call variant or not as part of the hashcode key.